apps/aeutils/priv/generate_poseidon.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+TS=(2 3 4 5 6 7 8 9)
+RP=(56 57 56 60 60 63 64 63)
+RF=8
+
+CURVE="BLS12_381"
+PRIME="0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001"
+SIZE=255
+
+# CURVE="BN128"
+# PRIME="0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000001"
+# SIZE=254
+
+SCRIPT=/Users/hans/Repos/hadeshash/code/generate_parameters_grain.sage
```

Comment:
Not a very portable path?

Reply:
Nah, let me replace that with a `git clone` - including the script is mainly for reference I don't expect anyone to run it 😅